lib/zipkin-tracer/zipkin_sqs_sender.rb

```diff
@@ -0,0 +1,29 @@
+require "aws-sdk-sqs"
+require "json"
+require "zipkin-tracer/zipkin_sender_base"
+require "zipkin-tracer/hostname_resolver"
+
+module Trace
+  class ZipkinSqsSender < ZipkinSenderBase
+    IP_FORMAT = :string
+
+    def initialize(options)
+      sqs_options = options[:region] ? { region: options[:region] } : {}
+      @sqs = Aws::SQS::Client.new(**sqs_options)
+      @queue_name = options[:queue_name]
+      @logger = options[:logger]
+      super(options)
+    end
+
+    def flush!
```

Comment:
This will be executed in the thread dealing with the request, maybe better to use the same Async mechanism than the http sender instead

Reply:
updated in https://github.com/openzipkin/zipkin-ruby/pull/147/commits/98c599b1d31c05f478136af504e8ae53dc7a9018